pyproject.toml

```diff
@@ -54,3 +54,41 @@ include = ["astroid*"]
 
 [tool.setuptools.dynamic]
 version = {attr = "astroid.__pkginfo__.__version__"}
+
+[tool.aliases]
+test = "pytest"
+
+[tool.pytest.ini_options]
+addopts = '-m "not acceptance"'
+python_files = ["*test_*.py"]
+testpaths = ["tests"]
+
+[tool.isort]
+include_trailing_comma = true
+known_first_party = ["astroid"]
+known_third_party = ["attr", "nose", "numpy", "pytest", "six", "sphinx"]
+line_length = 88
+multi_line_output = 3
+skip_glob = ["tests/testdata"]
+
+[tool.mypy]
+enable_error_code = "ignore-without-code"
+no_implicit_optional = true
+scripts_are_modules = true
+show_error_codes = true
+warn_redundant_casts = true
+
+[[tool.mypy.overrides]]
+# Importlib typeshed stubs do not include the private functions we use
```

Comment:
I would have preferred to keep this comment on L86 so we know what it is referring to